tools/topology/sof/pipe-src-volume-playback.m4

```diff
@@ -61,13 +61,13 @@ W_PGA(0, PIPELINE_FORMAT, 2, 2, playback_pga_conf, LIST(`		', "PIPELINE_ID Maste
 
 # Playback Buffers
 W_BUFFER(0, COMP_BUFFER_SIZE(3,
-	COMP_SAMPLE_SIZE(PIPELINE_FORMAT), PIPELINE_CHANNELS, SCHEDULE_FRAMES),
```

Comment:
Do we miss some setting to 
`W_VENDORTUPLES(media_src_tokens, sof_src_tokens, LIST(`		', `SOF_TKN_SRC_RATE_OUT	"PIPELINE_RATE"'))`
in this file?

Reply:
yes, you are right. As this PR has been lingering for almost a year, topologies come an go...